app/src/androidTest/java/com/reach5/identity/sdk/demo/SandboxSuccessTest.kt

```diff
@@ -0,0 +1,214 @@
+package com.reach5.identity.sdk.demo
+
+
+import android.view.View
+import android.view.ViewGroup
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.filters.LargeTest
+import androidx.test.rule.ActivityTestRule
+import androidx.test.runner.AndroidJUnit4
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers.`is`
+import org.hamcrest.Matchers.allOf
+import org.hamcrest.TypeSafeMatcher
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.*
+import java.util.regex.Pattern
+
+@LargeTest
+@RunWith(AndroidJUnit4::class)
+class SandboxSuccessTest {
+
+    @Rule
+    @JvmField
+    var mActivityTestRule = ActivityTestRule(MainActivity::class.java)
+    private fun anEmail(): String = UUID.randomUUID().let { uuid -> "$uuid@testaccount.io" }
```

Comment:
Are the created profiles deleted after the test ?

Reply:
is there a resource that allows the deletion of an account in our sdk ?

Reply:
It's not possible with UiTest. we agreed that we will give up the idea of ​​cleaning up after the fact and simply reset the environment from time to time

Reply:
The `org.junit.BeforeClass` and `org.junit.AfterClass` annotations should help with fixtures. It should be possible to have a cleanup method. We chose not to do the cleanup at this time because there is no profile deletion (management API) support in the SDK. Maybe this could be done by having the required support in the test module only ?